src/statistics.jl

```diff
@@ -142,6 +142,58 @@ function Base.show(io::IO, method::GeodesicInterpolationWithinRadius)
     return print(io, "GeodesicInterpolationWithinRadius($(method.radius))")
 end
 
+"""
+    Statistics.cov(
+        M::AbstractManifold,
+        x::AbstractVector;
+        basis::AbstractBasis=DefaultOrthonormalBasis(),
+        tangent_space_covariance_estimator::CovarianceEstimator=SimpleCovariance(;
+            corrected=false,
+        ),
+        mean_estimation_method::AbstractEstimationMethod=GradientDescentEstimation(),
+        inverse_retraction_method::AbstractInverseRetractionMethod=default_inverse_retraction_method(
+            M,
+        ),
+    )
+
+Estimate the covariance matrix of points `x` on manifold `M`. Since the covariance matrix
+on a manifold is a rank 2 tensor, the function returns its coefficients in basis induced by
+the given tangent space basis. See Section 5 of [^Pennec2006] for details.
+
+The mean is calculated using the specified `mean_estimation_method`, and tangent vectors at
+this mean are calculated using the provided `inverse_retraction_method`. Finally, the
+covariance matrix in the tangent plane is estimated using `tangent_space_covariance_estimator`
+(the type is defined in `StatsBase.jl` and examples of covariance estimation methods can be
+found in `CovarianceEstimation.jl`).
```

Comment:
Would it be reasonable to link both packages to their repositories or documentation urls?

Reply:
I also think that this sentence might be misleading: StatsBase/CovarianceEstimation does not define a tangent_space_covariance_estimator. It defines vector space ones and the tangent space is a vector space.

Reply:
OK, I'll rephrase it.